zubhub_frontend/zubhub/src/components/Navbar/Navbar.jsx

```diff
@@ -76,6 +76,8 @@ function NavBar(props) {
   const [searchType, setSearchType] = useState(getQueryParams(window.location.href).get('type') || SearchType.PROJECTS);
   const formRef = useRef();
   const token = useSelector(state => state.auth.token);
+  const pathname = props.location?.pathname
+  const hideSearchAndActions = pathname === '/signup' || pathname === '/login';
```

Comment:
@coderatomy is this part of the requirements? I think a user should be allowed access to the search box even in the signup page (since are user can search for things while not logged in)